### ISSUE TITLE
feature request: display icon based on bundleID

### ISSUE THREAD

**Issue by umiiii** [johnspurlock/app-privacy-report-viewer#2]
iTunes provided an API which allows to retrieve app icon via bundleID:

```
https://itunes.apple.com/lookup?bundleId=${bundleId}
```

So I wonder if we can stored the icon url when we initialize the database and use later.

```typescript
interface AppInfo {
  resultCount: number;
  results: Array<{
    artworkUrl60: string;
  }>
}

async function getIconByBundleId(bundleId: string): Promise<string> {
      const appStoreLookupUrl = `https://itunes.apple.com/lookup?bundleId=${bundleId}`
      const response = await fetch(appStoreLookupUrl)
      const appInfo: AppInfo = await response.json()
      if (appInfo.resultCount !== 1) {
        console.log('Can not determine AppStore ID: multiple or none results');
        console.log(appInfo);
        return Promise.reject('bad json')
      } else {
        console.log(appInfo.results[0].artworkUrl60);
        return Promise.resolve(appInfo.results[0].artworkUrl60);
      }
}
```

(Prototype, I am not very good at TypeScript :(  )

**Comment by johnspurlock** [comment]
I like this idea!  I'll look into integrating these icons in a way that is not too distracting.

**Comment by johnspurlock** [comment]
Ok, I added a basic icon fetcher in 70edb82862d1fb0a54fd6af04e5cb80ebcfbf14f.  Should be included in Release 0.5
